### PR TITLE
Make background transparent, remove resizing and set minimum size to 1x1 grid for camera widget

### LINKED ISSUE #3373: Allow the camera widget to resize based on image aspect ratio
**Is your feature request related to a problem? Please describe.**

It would help with the aesthetics - the camera widget could look more polished.
 
**Describe the solution you'd like**

Allow the camera widget to resize based on image aspect ratio so there are no black/grey bars and the widget looks more polished. 

**Describe alternatives you've considered, if any**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by dshokouhi**
## Summary

Should fix #3373 and #3374 by removing the hard coded resize options and allowing the image view to respect the aspect ratio. I also lowered the minimum widget size to account for low resolution images and made the background transparent.

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

File: app/src/main/res/xml/camera_widget_info.xml [hunk 1]
```diff
@@ -3,8 +3,8 @@
     android:configure="io.homeassistant.companion.android.widgets.camera.CameraWidgetConfigureActivity"
     android:initialKeyguardLayout="@layout/widget_camera"
     android:initialLayout="@layout/widget_camera"
-    android:minWidth="250dp"
-    android:minHeight="80dp"
+    android:minWidth="40dp"
```
Comment (jpelgrom) [inline app/src/main/res/xml/camera_widget_info.xml:6]: Should `android:targetCellWidth="2"`, `android:targetCellHeight="2"` be added so the widget keeps the current size in the launcher widget gallery? Allowing 1x1 is nice but by default it is quite small.

**Review by jpelgrom — COMMENTED** [review]
Not a big fan of inconsistent rounded corners but I guess this is the best we can do without resizing.

|    |Corner|Bounds|
|---|---|---|
|Partially rounded corner|image|image
|No rounded corner|image|image|

Resizing to at least `AppWidgetManager.OPTION_APPWIDGET_MAX_WIDTH` / `AppWidgetManager.OPTION_APPWIDGET_MAX_HEIGHT` without changing the aspect ratio, and allowing Android to downsize, would resolve this as we can set the widget size to wrap content, but I couldn't get it to work easily.

File: app/src/main/res/xml/camera_widget_info.xml (hunk 1 above)
Comment (dshokouhi) [reply app/src/main/res/xml/camera_widget_info.xml:6]: yes thats a great suggestion, thank you :)

**Review by jpelgrom — APPROVED** [review]

**Review by JBassett — APPROVED** [review]

**Comment by jpelgrom** [comment]
@Toukite Could you create an issue for that, including details? A closed PR is not the place to discuss this.

**Comment by Toukite** [comment]
@jpelgrom done https://github.com/home-assistant/android/issues/3443

**Pull request merged**